PosteAPI/views.py

```diff
@@ -83,6 +81,29 @@ def post(self, request, *args, **kwargs):
                 {"result": {"success": False, "errors": serializer.errors}},
                 status=status.HTTP_400_BAD_REQUEST,
             )
+          
+    def update(self, request, *args, **kwargs):
+        user = request.user
+        data = request.data
+        folder = Folder.objects.get(['folder_id'])
```

Comment:
I'm not sure this has the intended effect; do you mean `Folder.objects.get(id=data['folder_id'])` ? Might also be useful to include Swagger documentation to give insight into the desired request format (this has the added advantage of allowing us to test the endpoint via the built-in Swagger page at localhost:8000/swagger)